_posts/2021-10-01-selection-sort.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Selection Sort - A simple sorting algorithm"
+description: "This blog post is about Selection Sort, a simple sorting algorithm. Here, we will talk more about selection sort, how it works, and finally look at its pseudo-code for its implementation."
+keywords: 
+    - Mentro
+    - Selection Sort
+    - Sorting Algorithm
+category: dsa
+layout: post
+author: niyoj
+avatar: "/assets/images/author/niyoj.jpg"
+tags:
+    - sorting algorithm
+    - dsa
+    - selection sort
+permalink: /dsa/:title/
+image: /assets/images/b1/mentor.jpg
```

Comment:
For this either use the default image for DSA or upload a new image and link it.

Reply:
Ok will consider that too.